### DISCUSSION TITLE
Facing 419 | Page Expired even if CSRF present in the form

### DISCUSSION THREAD

**Discussion by mprince2k18** [laravel/framework#36904] [Q&A]
- Laravel Version: 8.35.0
- PHP Version: 7.4.0
- Database Driver & Version:

### Description:

Facing a CSRF issue when I deployed my project on a VPS server. Here is the video: https://www.loom.com/share/e2cd2c7ae2f6490ab1717d3243592513

Here is my form
![image](https://user-images.githubusercontent.com/43857625/113865389-d7d3bd80-97cd-11eb-83f5-db427a826598.png)

I have no idea why I am facing this.

Here is my application:
https://tukhulu.com/login
https://tukhulu.com/register

Please help

**Comment by ProteanCode** [comment]
Having same problem. Randomly out of nowhere I started to get a 419 mismatches...

**Comment by ProteanCode** [comment]
This had to have something related with cache

Try to call these commands:

```
        Artisan::call('optimize:clear');
        Artisan::call('view:clear');
        Artisan::call('route:clear');
        Artisan::call('optimize');
```

**Comment by endo64** [comment]
Same issue, see what I tried in this discussion: https://github.com/laravel/framework/discussions/39826
Still no luck 😢
